src/ApiCompat/ApiCompat.csproj

```diff
@@ -7,7 +7,6 @@
     <AssemblyName>ApiCompat</AssemblyName>
     <FileAlignment>512</FileAlignment>
     <DefineConstants>$(DefineConstants);COREFX</DefineConstants>
-    <NuGetRuntimeIdentifier>win7-x64</NuGetRuntimeIdentifier>
```

Comment:
No. even though this is an .exe, [we only copy the exe into the package](https://github.com/dotnet/buildtools/blob/master/src/nuget/Microsoft.DotNet.BuildTools.nuspec#L33) and not its dependencies, so it should be fine to have the runtime identifier be None for building. I only added this runtime identifier on the projects where we want to deploy the nuget dependencies to the package.